src/libostree/ostree-sepolicy.c

```diff
@@ -155,6 +159,92 @@ ostree_sepolicy_class_init (OstreeSePolicyClass *klass)
                                                         G_PARAM_READWRITE | G_PARAM_CONSTRUCT_ONLY));
 }
 
+#ifdef HAVE_SELINUX
+
+/* Find the latest policy file in our root and return its checksum. */
+static gboolean
+get_policy_checksum (char **out_csum,
+                     GCancellable *cancellable,
+                     GError **error)
+{
+  gboolean ret = FALSE;
+  const char *binary_policy_path = selinux_binary_policy_path ();
+  g_autofree char *bindir_path = g_path_get_dirname (binary_policy_path);
+  g_autoptr(GFile) bindir = g_file_new_for_path (bindir_path);
+  g_autoptr(GFile) best_child = NULL;
+  int best_version = 0;
+
+  /* The binary_policy_path includes the prefix of the basename, but without the
+   * period, e.g. "policy". */
+  g_autofree char *binfile_prefix = g_path_get_basename (binary_policy_path);
+  g_autofree char *binfile_no_fnmatch = g_strdup_printf ("%s.*[!0-9]*",
+                                                         binfile_prefix);
+
+  g_autoptr(GFileEnumerator) direnum =
+    g_file_enumerate_children (bindir, OSTREE_GIO_FAST_QUERYINFO,
```

Comment:
FWIW I've been trying to avoid using `GFile` for newer code but I know there's a lot of it left around here in the selinux bits.  I'm okay with this but we should keep in mind it should be ported later.